### DISCUSSION TITLE
해당 화면이 나오면 정상 작동중인게 맞는걸까요?

### DISCUSSION THREAD

**Discussion by Karina2122** [changdoc/naver-vaccine-macro#55] [Q&A] [answered]
![K-057](https://user-images.githubusercontent.com/87792181/126583291-928ca1bb-1737-45af-8a39-cfad32f3412f.png)

해당 화면에서 계속 새로고침이 되고 있으면 정상 작동 되고있는게 맞을까요 ?

인증서 발급은 해두었는데,

'네이버 인증서로 인증' 이란 말이 인증이 되지 않았단 얘기인가 싶어 질문글 올립니다.

P.S. 예약 성공 여부와 관계없이 개발자님, 저에게도 기회를 갖게 해주셔서 정말 감사드립니다 :)

**Comment by soosooho** [comment, marked as answer]
아직 인증 안된것같은데요.. 휴대폰으로 인증하고 넘어가보세요

**Comment by changdoc** [comment]
네 인증이 안된 상태네요. 인증 진행하시고 매크로 동작 시키셔야 합니다.

**Comment by Karina2122** [comment]
답변 해주신 분들 감사드립니다. 이상하게 크롬에서는 인증서고 핸드폰 인증이고 반응이 없었는데, 웨일로 하니 잘됩니다. 감사해요 :)
